### ISSUE TITLE
Bug: Shouldn't close video tabs

### ISSUE THREAD

**Issue by cantonalex** [bonafideduck/CloseTabsExtension#3]

**Comment by bonafideduck** [comment]
Any pinned tab will never be removed.  Will that solve your problem?

Typically, if I care about a video, it will be in the foreground anyway and not be removed.  Also, if minimum tabs apply to each window, then a video playing in a second window shouldn't be closed either.

**Comment by cantonalex** [comment]
What about podcasts? or music in a tab? I have been using this since yesterday. I normally use Tab wrangler in Chrome. Currently yours is just closing podcast windows mid sentence as well as videos. Very frustrating.

If a sound is coming from a tab. It should never close it regardless of being pinned or not. People don't pin a tab just to watch a video or listen to a podcast.

**Comment by cantonalex** [comment]
@bonafideduck I was mentioning Tab wrangler because it is a model of how yours should work. I have had to switch back to Tab Wrangler because it doesn't close mid video on an active tab like your app does.

**Comment by bonafideduck** [comment]
@cantonalex  I'm considering a different route.  I'm going to look into replacing CloseTabs for Safari with a port of TabWrangler.  Were you trying CloseTabs because it was the only thing available on Safari (DuckTabs on Chrome) or was there a capability in it that would be nice in tab wrangler?

**Comment by cantonalex** [comment]
Yes I tried it because there was nothing else available for safari
